security/rbac/delta_configs/connector-source.properties.delta

```diff
@@ -0,0 +1,3 @@
+producer.override.sasl.mechanism=OAUTHBEARER
```

Comment:
Following up on comments left on the `connect-avro-distributed.properties.delta` file--we should remove the `producer.override.sasl.mechanism` and `producer.override.sasl.login.callback.handler` configs from this file and they should instead be set in the worker file.

Reply:
Agree